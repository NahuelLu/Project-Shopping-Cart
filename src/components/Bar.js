import '../styles/Bar.css'
const Bar = ({amount})=>{

    return (
        <nav className='sticky-bar'>
            <h4>Items selected: {amount}</h4>
        </nav>
    )
}

export default Bar