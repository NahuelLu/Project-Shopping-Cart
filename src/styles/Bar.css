.sticky-bar{
    display: flex;
    align-items: center;
    width: 100%;
    justify-content: end;
    gap: 20px;
}