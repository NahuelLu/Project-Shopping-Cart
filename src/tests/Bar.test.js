import { render, screen } from "@testing-library/react"
import Bar from "../components/Bar"
describe("Test for Bar component",()=>{

    test("Renders successfully",()=>{
        const {container}= render(<Bar/>)
        expect(container).toMatchSnapshot()
    })
    test("Renders succesfully 20 amount in heading",()=>{
        const amount= 20
        const {container} = render(<Bar amount={amount}/>)
        expect(container).toMatchSnapshot()
    })
})
