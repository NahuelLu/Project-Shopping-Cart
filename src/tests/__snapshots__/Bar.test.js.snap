// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for Bar component Renders succesfully 20 amount in heading 1`] = `
<div>
  <nav
    class="sticky-bar"
  >
    <h4>
      Items selected: 
      20
    </h4>
  </nav>
</div>
`;

exports[`Test for Bar component Renders successfully 1`] = `
<div>
  <nav
    class="sticky-bar"
  >
    <h4>
      Items selected: 
    </h4>
  </nav>
</div>
`;
